style.css

```diff
@@ -394,3 +395,118 @@ p {
   list-style-type: none;
   text-decoration: none;
 }
+
+.form-section {
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  text-align: center;
+}
+
+.form-section > h2 {
+  margin: 120px 24px 73px 24px;
+  position: static;
+  height: 216px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.inputs {
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  list-style: none;
+}
+
+.input1 {
+  margin: 0 40px 0 40px;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  color: #979493;
+  display: flex;
+}
+
+.email {
+  margin-top: 25px;
+}
+
+.separator1 > div {
+  border: 1px solid #dbd8d7;
+  margin: 14px 24px 24px 24px;
+}
+
+.input1 > input {
+  padding-right: 100px;
+  border: none;
+  width: 295px;
+}
+
+.input2 {
+  margin: 0 24px 24px 24px;
+}
+
+.text-area {
+  background-color: #fbf8f7;
+  border: none;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  padding: 12px;
+  width: 303px;
```

Comment:
Our web page should be responsive to different types and sizes of screen devices.

- [ ] Please remove the fixed width, and set the width as wide as of the inputs; :+1: